src/main.js

```diff
@@ -0,0 +1,92 @@
+//query selectors / global variables
+var body = document.querySelector('body');
+var inputTitle = document.querySelector('.idea-form-title');
+var inputBody = document.querySelector('.idea-form-body');
+var ideaCardsGrid = document.querySelector('.idea-cards');
+var ideaForm = document.querySelector('form');
+var saveButton = document.querySelector('.idea-form-button');
+var hamburgerIcon = document.querySelector('.hamburger-icon');
+var menuCloseIcon = document.querySelector('.menu-close-icon');
+var menuDropDown = document.querySelector('.menu-dropdown')
+var blanket = document.querySelector('.blanket')
+
+var list = [];
+//eventlisteners
+body.addEventListener('click', clickHandler);
+body.addEventListener('keyup', keyupHandler);
+window.onload = disableEnableButton();
+//functions
+function clickHandler(event){
+    if(event.target.classList.contains('idea-form-button')){
+      saveIdea();
+    }
+    if (event.target.classList.contains('hamburger-icon')){
+      displayDropDown();
+    }
+    if(event.target.classList.contains('menu-close-icon')){
+      closeDropDown();
+    }
```

Comment:
Great find and debug on using classList.contains to properly target elements that have multiple class names assigned to them! This made it a lot easier to find the class we wanted without having to manipulate code separately to sort it out.